### ISSUE TITLE
400 Bad Request // Incorrect Username or Password

### ISSUE THREAD

**Issue by mishyy** [samczsun/Skype4J#98]
Hey,

When i try to join a skype chat using a guest account, using the correct chat id, obtained from /get name
On a normal, non-moderated chat, i receive this error:

hastebin.com/megeqivimu.tex

I am using the latest version of skype, and the api, in the .jar form, on IntelliJ 2016.1.2
Java 8, Windows 10

Also, when attempting to join a chat using a username and password account, it does not work, it says the details are in-correct, despite, what it is saying.

**Comment by mishyy** [comment]
Edit: I only obtain this error, when using the skype.subscribe() method.
Yes, it is in a try and catch

**Comment by ducc** [comment]
Mind sending your code?

**Comment by mishyy** [comment]
Yeah, sure; https://gist.github.com/Cmonster-11/cfc36ee62a4a7b6888f8ea7655e461a2
This contains all of the relevant project files.

**Comment by ctian1** [comment]
You didn't submit your username and password..

**Comment by mishyy** [comment]
That one is for using Guest accounts, i also tried using the Normal one.

**Comment by ducc** [comment]
How does this even compile? 
https://gist.github.com/Cmonster-11/cfc36ee62a4a7b6888f8ea7655e461a2#file-trollbot_src_xyz_cmonster_trollbot_commandlogout-java-L19

Does it work for guest users & if not are you able to use a guest account by using your browser?
